docs/setup_help/uta_installation.md

```diff
@@ -0,0 +1,33 @@
+1. Switch to your admin account #Skip if your user already has sudo permissions
```

Comment:
Perhaps we should put these with UTA? 

Also, I cringe. I/we need to find time to update and make it easier to install.

Reply:
I am happy to put these directions wherever. Katie in our lab was working on a master readme also. Since UTA is required for this to work it made sense in my head to include it here also. I am happy to be overridden on this though.

Reply:
I think it is okay to put here for now–don't want this blocking other content in PR. Let's make an issue at UTA to move these instructions over at some point, perhaps as part of an install cleanup.